pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1387,6 +1389,7 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundListenerForPortOrUDS(n
 		currentListenerEntry.protocol = protocol.Unsupported
 		currentListenerEntry.listener.ListenerFilters =
 			append(currentListenerEntry.listener.ListenerFilters, &listener.ListenerFilter{Name: envoyListenerHTTPInspector})
+		setListenerFiltersTimeoutIfNotSet(node, currentListenerEntry.listener)
```

Comment:
remove all of this.. we can always have the listener filter timeout. It wont hurt.